config.js

```diff
@@ -64,6 +65,7 @@ exports.updateFromArgv = function() {
         .describe('logfile', 'In addition to logging to the console, also log gtp2ogs output to a text file')
         .describe('json', 'Send and receive GTP commands in a JSON encoded format')
         .describe('kgstime', 'Set this if bot understands the kgs-time_settings command')
+        .describe('showboard', 'Set this if bot understands the showboard GTP command and you want to display the GTP showboard output')
```

Comment:
The description could be shortened to "Set this to issue showboard GTP command on every move".

Reply:
@windo 

while your sentence is shorter, it is more complicated to understand (for example the word "issue" does not have a simple meaning here)

i wanted to highlight that even if the bot supports showboard command, bot admin can still disabled it if he wants

i think we can shorten it like this (we lose a bit of clear sense, but it should be easy to guess, and can be detailed in OPTIONS-list)

`and you want to display it`

Reply:
because by `display it` it may mean : 
- display the GTP command text "showboard"
- or display the GTP showboard output

so i'll try to think a little more if there a short and clear way to write it

Reply:
also we may add a coma at the middle of the sentence, and use "and if" to make it clearer too